packages/payments/package.json

```
@@ -1,51 +1,52 @@
 {
   "name": "@graphprotocol/payments",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Payment Management",
+  "description": "State Channel Payment Manager",
```

Unintentional change:
```suggestion
"description": "Graph Network State Channel Payment Management",
```